plugins/commander.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: commander
+spec:
+  homepage: https://github.com/schabrolles/kubectl-commander
+  shortDescription: Interactively manage Kubernetes resources
+  version: "v0.4.0"
+  description: |
+    kube-commander is an interactive menu (based on fzf) that allows you
```

Comment:
name is not matching plugin name (make sure to use the `kubectl commander` syntax in your program, too).

you can fix this in a subsequent PR